robottelo/cli/factory.py

```diff
@@ -2578,17 +2578,17 @@ def vm_setup_ssh_config(vm, ssh_key_name, host, user=None):
     # setup the config file
     ssh_config_file_path = f'{ssh_path}/config'
     result = vm.run(f'touch {ssh_config_file_path}')
-    if result.return_code != 0:
+    if result.status != 0:
```

Comment:
I'll be following this PR very closely with #8665, which will remove VirtualMachine and ensure all of these uses are through ContentHost.